packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -243,7 +243,15 @@ abstract class FlutterCommand extends Command<Null> {
 
   /// The path to send to Google Analytics. Return null here to disable
   /// tracking of the command.
-  Future<String> get usagePath async => name;
+  Future<String> get usagePath async {
+    if (parent is FlutterCommand) {
+      final FlutterCommand commandParent = parent;
+      final String path = await commandParent.usagePath;
+      return '$path/$name';
```

Comment:
What does this do when commandParent.usagePath == null?

https://github.com/flutter/flutter/blob/c5a5945e92d01c671d34fb91aa4015170185c31a/packages/flutter_tools/lib/src/commands/config.dart#L62

Looks like documenting child commands of 'config' could be interesting here.

Reply:
I think dropping reports for parent nodes that are null seems reasonable for now -- defaulting to not reporting subcommands is the safe option.  We can come up with something more complicated should this ever come up.

Reply:
Makes sense; I updated this to not report for parents who return null for `usagePath`.